### ISSUE TITLE
nunit console run fails on asp.net core project file.

### ISSUE THREAD

**Issue by mrjpeterj** [nunit/vs-project-loader#27]
dotnet version 2.1.202
nunit console runner 3.8.0

csproj file start looks like this
```
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>net461</TargetFramework>
  </PropertyGroup>
```

and thus this line decides that it should be looking for `bin\Release\net461\<ProjectName>.dll` but ASP.Net Core projects build to .exe files.

https://github.com/nunit/vs-project-loader/blob/7fe84e90e4013d4d3230d48de5d35e603c40198b/src/extension/VSProject.cs#L253

Therefore I end up with.

```
1) Invalid : E:\jenkins\workspace\WebSites\DataDelivery\WebServer\APIServer\bin\Release\net461\APIServer.dll
File not found: E:\jenkins\workspace\WebSites\DataDelivery\WebServer\APIServer\bin\Release\net461\APIServer.dll
```

The project doesn't even have an tests in it, it's just in the sln file that contains projects that have tests in them, so it's rather frustrating that this breaks the unit tests part of the build

**Comment by ChrisMaddock** [comment]
Thanks for filing this bug @mrjpeterj! Would you be interested in contributing a pull request, to fix it?

In my opinion, the long term solution to these problems is to do https://github.com/nunit/vs-project-loader/issues/25 - but I definitely think we should keep patching the existing code, until someone volunteers to do #25.

> The project doesn't even have an tests in it, it's just in the sln file that contains projects that have tests in them, so it's rather frustrating that this breaks the unit tests part of the build

FYI - I _think_ a solution file which has non-test projects in it currently will fail the test run anyway. We've definitely talked about allowing a solution to contain non-test projects and this passing - I can't remember the conclusion however, I'm afraid!

**Comment by mrjpeterj** [comment]
nunit console runner 3.7.0 would successfully process a solution file with non-test projects in it, but it was failing to parse the .Net Core csproj, so I updated to 3.8.0 which did indeed fail the test run until I added `--skipnontestassemblies` to the command line.  
  
I will certainly consider trying to come up with a fix and submitting it if I can get an nunit dev environment up and running easily enough.

**Comment by ChrisMaddock** [comment]
Great - in that case, I must be wrong - sorry! And brilliant - let us know if you have any issues getting the dev environment set up! Hopefully it should just be a simple pull and build!

**Comment by ChrisMaddock** [comment]
Fixed by #28
